admin_guide/topics/about-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='about-cluster-auto-scaler-{context}']
+= About the {product-title} auto-scaler
+
+The auto-scaler in {product-title} periodically, such as every
```

Comment:
can we be more crisp here?  rather than state such as ...

Reply:
@frobware, do you have a crisper value for the interval?